certbot-nginx/certbot_nginx/parser.py

```diff
@@ -561,7 +588,8 @@ def find_location(direc):
     directive_name = directive[0]
     def can_append(loc, dir_name):
         """ Can we append this directive to the block? """
-        return loc is None or (isinstance(dir_name, str) and dir_name in REPEATABLE_DIRECTIVES)
+        return loc is None or (isinstance(dir_name, six.string_types)\
```

Comment:
nit: You don't need to escape the newline here.

Reply:
nice